roles/config-rh-sso/tasks/realm.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: "Set realm facts for realm: {{ item.name }}"
+  set_fact:
+    realm_id: "{{ item.id | default(item.name) }}"
+    realm_name: "{{ item.name }}"
+    realm_displayName: "{{ item.displayName | default(item.name) }}"
+    realm_enabled: "{{ item.enabled | default(true) }}"
+    realm_sslRequired: "{{ item.sslRequired | default('external') }}"
+    realm_registrationAllowed: "{{ item.registrationAllowed | default(true) }}"
+    realm_emailLogin: "{{ item.emailLogin | default(true) }}"
+    realm_dupsAllowed: "{{ item.dupsAllowed | default(false) }}"
+    realm_resetPwd: "{{ item.resetPwd | default(false) }}"
+    realm_editUname: "{{ item.editUname | default(false) }}"
+    realm_bruteForceProtected: "{{ item.bruteForceProtected | default(true) }}"
+
+- name: "Retrieve template for realm: {{ item.name }}"
+  set_fact:
```

Comment:
You can just call the `lookup` as part of line 25 below (i.e.: `body: "{{ lookup('template', ....`) to avoid another set_fact here.